### PR TITLE
Latest HydraDX / Basilisk

### PR DESCRIPTION
**Pull request by nohaapav**
Hi guys, i've fixed some HydraDX & Basilisk stuff regarding v3, also added new adapters .. Feel free to merge

### CODE REVIEW

**Comment by nohaapav** [comment]
Guys removed dest weight from hydradx route config again, we went to v3 yesterdy
https://github.com/polkawallet-io/bridge/pull/69/commits/6b33c9ad791cdb1d18e148a1709b68444fdbd277

File: src/adapters/hydradx.ts [hunk 1]
```diff
@@ -35,6 +43,13 @@ export const basiliskRouteConfigs = createRouteConfigs("basilisk", [
       fee: { token: "KUSD", amount: "5060238106" },
     },
   },
+  {
+    to: "karura",
+    token: "aUSD",
```
Comment (RomeroYang) [inline src/adapters/hydradx.ts:48]: No need here either.

File: src/adapters/hydradx.ts [hunk 2]
```diff
@@ -87,13 +123,41 @@ export const basiliskTokensConfig: Record<string, ExtendedToken> = {
     ed: "10000000000",
     toRaw: () => 2,
   },
+  aUSD: {
```
Comment (RomeroYang) [inline src/adapters/hydradx.ts:126]: No need here either.

File: src/configs/chains/kusama-chains.ts [hunk 3]
```diff
@@ -139,4 +139,20 @@ export const kusamaChains = {
...
+    type: "substrate",
+    icon: "https://resources.acala.network/_next/image?url=%2Fnetworks%2Frobonomics.png&w=96&q=75",
+    paraChainId: 2048,
+    ss58Prefix: 32,
+  },
+  tinkernet: {
+    id: "tinkernet",
+    display: "Tinkernet",
+    type: "substrate",
+    icon: "",
```
Comment (RomeroYang) [inline src/configs/chains/kusama-chains.ts:154]: It's better to provide an icon source, you may add the icon into resouces.acala.network through this repo: https://github.com/open-web3-stack/token-resources

**Review by RomeroYang — CHANGES_REQUESTED** [review]
1. remove unused token aUSD in Karura, and use KUSD instead.
2. provide an network icon for tinkernet (optional)

**Comment by tien** [comment]
Awesome stuff @nohaapav & @RomeroYang, can't wait for this to get merged 💪

File: src/adapters/hydradx.ts (hunk 2 above)
Comment (nohaapav) [reply src/adapters/hydradx.ts:125]: Done

File: src/adapters/acala/karura-configs.ts [hunk 4]
```diff
@@ -324,13 +324,27 @@ export const karuraRouteConfigs = createRouteConfigs("karura", [
...
   {
     to: "basilisk",
     token: "KUSD",
     xcm: {
       fee: { token: "KUSD", amount: "3150402683" },
     },
   },
+  {
+    to: "basilisk",
+    token: "aUSD",
```
[the remark this answers is not in the document]
Comment (nohaapav) [reply src/adapters/acala/karura-configs.ts]: Done

**Comment by nohaapav** [comment]
@RomeroYang @tien Guys btw fixed interlay (they went to v3 21st of June), also aUSD has been removed. Plus additional centrifuge route was added for hydradx. ;) So just tinkernet icon is missing but i guess not a blocker atm ..

**Review by RomeroYang — APPROVED** [review]

**Pull request merged**